src/arch/arm/subarch/armv7a/cpu_reset.S

```diff
@@ -13,6 +13,14 @@
 
 .global hardware_init_hook
 hardware_init_hook:
+#ifdef CORTEXA7_MULTIPROCESSING
```

Comment:
Thanks a lot.
Could You use `OPTION_GET(NUMBER, xxx)` (and option in Mybuild)  instead of setting `-DCORTEXA7_MULTIPROCESSING` in CFLAGS, to determine whether multicore use or not